edx_proctoring/api.py

```diff
@@ -205,6 +205,19 @@ def get_review_policy_by_exam_id(exam_id):
     return ProctoredExamReviewPolicySerializer(exam_review_policy).data
 
 
+def _get_review_policy_by_exam_id(exam_id):
+    """
+    Looks up exam by the primary key. Returns None if not found
+
+    Returns review_policy field of the Django ORM object
+    """
+    try:
+        exam_review_policy = get_review_policy_by_exam_id(exam_id)
+        return ProctoredExamReviewPolicySerializer(exam_review_policy).data['review_policy']
```

Comment:
Yup, I did test it in the sandbox and it seems to work. If there is no exam policy `get_review_policy_by_exam_id()` will throw an exception, so we won't get to that first return statement in this case. Because of that it's not an issue that `review_policy` defaults to an empty string.